client/jetpack-connect/remote-credentials.js

```diff
@@ -202,9 +201,6 @@ export class OrgCredentialsForm extends Component {
 					</div>{ ' ' }
 					{ removedProtocolURL }
 				</div>
-				<div className="jetpack-connect__wordpress-logo">
```

Comment:
There are some styles associated with the `jetpack-connect__wordpress-logo` classname that we can now remove.